package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "project-leaderboard",
+  "version": "1.0.0",
+  "description": "a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules.",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
Great job setting up webpack for your project👍. However, you forgot to add a script to start the webpack server for development. You can do this by adding `"start": "webpack serve --open"`  to your scripts in your package.json file.🙏. This way, its easy to startthe webpack server

Reply:
This change has been done well